DotNetCliPerf/CoreApp.cs

```diff
@@ -11,7 +11,7 @@ public abstract class CoreApp : DotNetApp
     {
         private const string _globalJson = @"{ ""sdk"": { ""version"": ""0.0.0"" } }";
 
-        [Params("2.0.2", "2.1.4", "2.1.300-preview2-008012")]
+        [Params("2.0.2", "2.1.4", "2.1.300-preview2-008012", "2.2.0-preview1-007947")]
```

Comment:
IIRC this SDK is actually **older** than `2.1.300...` - they changed their numbering.

Reply:
Alright, I'll revert this change.

Reply:
Yes, I update the list frequently as I test new SDKs, so I recommend not changing this in the PR.